src/azure-cli/azure/cli/command_modules/backup/_format.py

```diff
@@ -43,14 +43,10 @@ def transform_job(result):
     columns.append(('Operation', result['properties']['operation']))
     columns.append(('Status', result['properties']['status']))
     columns.append(('Item Name', result['properties']['entityFriendlyName']))
+    columns.append(('Backup Management Type', result['properties']['backupManagementType']))
     columns.append(('Start Time UTC', result['properties']['startTime']))
-
-    if result['properties']['backupManagementType'] == 'AzureIaasVM':
-        columns.append(('Duration', result['properties']['duration']))
-    elif result['properties']['backupManagementType'] == 'AzureStorage':
-        columns.append(('Duration', result['properties']['additionalProperties']['duration']))
-    elif result['properties']['backupManagementType'] == 'AzureWorkload':
-        columns.append(('Duration', result['properties']['duration']))
+    columns.append(('Duration', "0:00:00.000000"))
+    columns[6] = ('Duration', result['properties']['duration'])
```

Comment:
Don't write static index while assigning the value. you can rewrite the same thing using 
duration = "0:00:00.00000"
if result['properties']['duration'] is not None:
   duration = result['properties']['duration']
columns.append('Duration', duration)